CRM/Contact/Form/Edit/EmailBlockTrait.php

```diff
@@ -117,7 +149,6 @@ protected function addEmailBlockContactFields(int $blockNumber): void {
         }",
     ];
     $this->addElement('radio', "email[$blockNumber][is_primary]", '', '', '1', $js);
-    $this->addCustomDataFieldBlock('Email', $blockNumber);
```

Comment:
@eileenmcnaughton is this removal right? won't that affect the contact summary screens?

Reply:
@seamuslee001 I moved it from the contact fields to the non contact fields - both are included on contact summary cos it calls `getEmailBlockFields()`

![image](https://github.com/user-attachments/assets/34a5a89b-9194-442d-aecf-89e49b803fe1)